File: src/main/java/com/sedmelluq/discord/lavaplayer/container/mpeg/reader/MpegSectionHandler.java
package com.sedmelluq.discord.lavaplayer.container.mpeg.reader;

import java.io.IOException;

/**
 * Handles one MPEG section which has no version info
 */
public interface MpegSectionHandler {
  /**
   * @param child The section
   * @throws IOException On read error
   */
  void handle(MpegSectionInfo child) throws IOException;
}

File: main/src/main/java/lavaplayer/container/mpeg/reader/MpegSectionHandler.kt
package lavaplayer.container.mpeg.reader

import java.io.IOException

/**
 * Handles one MPEG section which has no version info
 */
interface MpegSectionHandler {
    /**
     * @param child The section
     * @throws IOException On read error
     */
    @Throws(IOException::class)
    fun handle(child: MpegSectionInfo)
}
